ts/src/base/Exchange.ts

```diff
@@ -3566,16 +3566,15 @@ export default class Exchange {
     }
 
     parseDepositAddresses (addresses, codes = undefined, indexed = true, params = {}) {
-        let result = undefined;
+        let result: any[] | {} = [];
```

Comment:
@samgermain this annotation cannot be transpiled, we can probably get way with `any` here, I will take a look